server/controllers/availability.js

```diff
@@ -0,0 +1,140 @@
+const Availability = require("../models/Availability");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+
+// @route POST /availability
+// @desc create a schedule
+// @access private
+exports.createSchedule = asyncHandler(async (req, res, next) => {
+    console.log(req)
+    console.log(req.body)
+    console.log(req.body.schedule)
+
+    console.log(req.user)
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. :smiley: